sdk/go/pulumi/transform.go

```diff
@@ -47,3 +47,31 @@ type ResourceTransformResult struct {
 // of the original call to the `Resource` constructor.  If the transform returns nil,
 // this indicates that the resource will not be transformed.
 type ResourceTransform func(context.Context, *ResourceTransformArgs) *ResourceTransformResult
+
+// InvokeTransformArgs is the argument bag passed to a invoke transform.
+type InvokeTransformArgs struct {
+	// The token of the invoke.
+	Token string
```

Comment:
Is "token" a concept that is in the "user-facing API" as it stands today? If not, should we think about how this is presented/give examples so that users know what to expect from this field? Or potentially there's a neater thing to expose, etc.

Reply:
Good question.  I'm not sure if there's anything better we can do here.  The token is what's passed as first argument to the Invoke, but I don't think we're exposing this to the user right now.  Not sure of anything better we could pass here though.

Reply:
It's similar to "Type" in the resource transform. A little less user facing than "Type" already is today, but I agree with Thomas's assessment that there isn't anything better.